integration-tests/chopsticks/src/tests/xcm/__snapshots__/limitedReserveTransferSpiritnetHydraDx.test.ts.snap

```diff
@@ -0,0 +1,147 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`Limited Reserve V2 Transfers from Spiritnet Account Bob -> HydraDx > receiver events currencies 1`] = `
+[
+  {
+    "data": {
+      "amount": 100000000000000000,
+      "currencyId": 60,
+      "who": "7L53bUTBbfuj14UpdCNPwmgzzHSsrsTWBHX5pys32mVWM3C1",
+    },
+    "method": "Deposited",
+    "section": "currencies",
+  },
+]
+`;
+
+exports[`Limited Reserve V2 Transfers from Spiritnet Account Bob -> HydraDx > receiver events xcmpQueue 1`] = `
+[
+  {
+    "data": {
+      "messageHash": "(hash)",
+      "messageId": "(hash)",
+      "weight": {
+        "proofSize": 0,
+        "refTime": 400000000,
+      },
+    },
+    "method": "Success",
+    "section": "xcmpQueue",
+  },
+]
+`;
+
+exports[`Limited Reserve V2 Transfers from Spiritnet Account Bob -> HydraDx > sender events Balances 1`] = `
+[
+  {
+    "data": {
+      "amount": "(rounded 170000000000)",
+      "who": "4seWojfEHrk5YKPahdErazQ3CWEHZYi6NV4gKz5AaejWbRPJ",
```

Comment:
This is the address of Alice

Reply:
Shouldn't it be Bob's?

Reply:
No, please take a look here: https://github.com/KILTprotocol/kilt-node/pull/614/files#diff-ce4305e382897732899a9335dad6b6d4350a23b2d2be29fc5c4ca6f864806a48R31-R39

Alice is transmitting the extrinsic. The balance has to be withdrawn from her right?

Reply:
The snapshot says "Limited Reserve V2 Transfers from Spiritnet Account Bob -> HydraDx > sender events Balances 1".

Reply:
True sorry: [71a810b](https://github.com/KILTprotocol/kilt-node/pull/614/commits/71a810bfdfdbdaf7b1a2022f283f59036e790a16)